plugins/module_utils/system_access_users_utils.py

```diff
@@ -39,70 +39,114 @@
     xml_utils,
     opnsense_utils,
 )
+
 from ansible_collections.puzzle.opnsense.plugins.module_utils.config_utils import (
     OPNsenseModuleConfig,
 )
-from ansible_collections.puzzle.opnsense.plugins.module_utils.enum_utils import ListEnum
 
 
-class OPNSenseGroupNotFoundError(Exception):
+class OPNSenseCryptReturnError(Exception):
```

Comment:
```suggestion
class OPNsenseCryptReturnError(Exception):
```

Reply:
Thanks for the suggestion, fixed in 2ee63a3.